trio/_ssl.py

```diff
@@ -207,15 +214,19 @@ class NeedHandshakeError(Exception):
 
     """
 
+    __slots__ = ()
+
 
 class _Once:
-    def __init__(self, afn, *args):
+    __slots__ = ("_afn", "_args", "started", "_done")
+
+    def __init__(self, afn: Callable[..., Awaitable[Any]], *args: Any) -> None:
```

Comment:
I think `object` can be used for `*args`, and we don't care about the result of `afn` so just `Awaitable[object]` is AOK.

Reply:
Changed in edbf981